### PR TITLE
updates load-testing notebook

### PR DESCRIPTION
**Pull request by charlesfrye**
Awesome notebook Sayak! I learned a lot. Below is a summary of my edits.

I gave the text a pass for style and clarity.

I also simplified the presentation a bit:

1. I dropped the writing of the config file, since you can just pass CLI args instead. There's a note that points out that using a config file is the better choice long-term.
2. We now display the HTML report inside the notebook.
3. I simplified the plotting by using the pandas built-in plotting API.

### CODE REVIEW

File: notebooks/lab99_loadtesting.ipynb [hunk 1]
```diff
@@ -4,7 +4,9 @@
   "metadata": {
```
Comment (sayakpaul) [inline notebooks/lab99_loadtesting.ipynb:4]: <p>Seems like we only need to <code># Writing a load test with locust. </code> ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sayakpaul/fsdl-text-recognizer-2022/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

**Comment by sayakpaul** [comment]
Thanks @charlesfrye!

Just one minor comment and I think we should be good to go.

File: notebooks/lab99_loadtesting.ipynb (hunk 1 above)
Comment (charlesfrye) [reply notebooks/lab99_loadtesting.ipynb:4]: What's the issue? I see it rendering weirdly in review NB but the raw JSON and the actual notebook look fine to me.

Comment (sayakpaul) [reply notebooks/lab99_loadtesting.ipynb:4]: Sorry. All good.

**Pull request merged**